sale_validity/model/sale_order.py

```diff
@@ -34,7 +34,7 @@ class sale_order(orm.Model):
             readonly=True,
             states={
                 'draft': [('readonly', False)],
-                'sent': [('readonly', True)],
+                'sent': [('readonly', False)],
```

Comment:
Oups sorry, this commit was for our internal Odoo only.

I'll remove this commit from the pull-request ASAP.